src/frontend/scss/generic/_accessibility.scss

```diff
@@ -28,6 +24,9 @@
   background-color: r-color('midnightblue');
   padding: 1rem;
   z-index: 100000;
+  position: absolute;
```

Comment:
Is `fixed` better in this case ?

Reply:
Because it should only appear if keyboard foccused, `absolute` should be the way to go. (IMHO)

Reply:
It doesn't change much actually. Right now `absolute` is kinda better because it doesn't mess with the django toolbar when editing a page. But for the actual use-case without the toolbar, since the link is the first on the page, shown at the very top, absolute or fixed, whatever.